calibration/METBKA/CGINS-METLGR-00091__20150507.csv

```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR033,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP01CNSM-00003 CP03ISSM-00002 CP04OOSM-00002
+LGR033,CC_height_of_air_humidity_measurement_m,4.255,
+LGR033,CC_height_of_air_temperature_measurement_m,4.25,
+LGR033,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```

Comment:
Logger SN should match filename, filename SN does not match Bulk Load SN CGINS-METLGR-99991, CC_height_of_air_temperature_measurement_m should be 4.255, CP04OOSM should be CP04OSSM

Reply:
Agree w Allen - assume that this UID should match the addition in sensor bulk load & should be METLGR-99991; filename and S/N column in csv need to be fixed; fix height of air temp cal coeff; fix mooring id.